scripts/container-only/wkdev-setup-vscode

```diff
@@ -17,8 +17,10 @@ verify_executables_exist curl
 argsparse_allow_no_argument true
 argsparse_use_option "=yes" "Assume yes for all prompts."
 argsparse_use_option "no-extensions" "Don't install extensions."
+argsparse_use_option "proprietary" "Use proprietary VSCode."
```

Comment:
I understand people have strong opinions on this but I would prefer the default being normal vscode.

I can accept vscodium as an opt-in option but the reality is it will be less tested.